bindings/bindingtester/tests/api.py

```diff
@@ -601,8 +601,10 @@ def generate(self, args, thread_number):
             elif op == 'TENANT_CLEAR_ACTIVE':
                 instructions.append(op)
             elif op == 'TENANT_LIST':
-                instructions.push_args(self.random.random_int())
                 self.ensure_string(instructions, 2)
+                instructions.push_args(self.random.random_int())
+                test_util.to_front(instructions, 2)
+                test_util.to_front(instructions, 2)
```

Comment:
What was the reason for doing it this way instead of the previous version you had that pushed the limit first?

Reply:
It was mostly to match the way that getrange was doing it, but I also think the previous way was also failing tests. I will have to double check

Reply:
Got it, I guess this is so that we can use any strings that are on top of the stack.